pydantic/types.py

```diff
@@ -2600,9 +2602,15 @@ class ThanksgivingDinner(BaseModel):
     """
 
     discriminator: str | Callable[[Any], Hashable]
+    """The callable to use to extract the value of the discriminator from the input."""
```

Comment:
Should probably explain what it means if it's a `str` — i.e., the name of the field to use as the discriminator, same as in the `Field` function.

Reply:
Done, thanks for catching that.